src/test/java/com/softserveinc/ita/vsaroz/SignUpTest.java

```diff
@@ -0,0 +1,45 @@
+package com.softserveinc.ita.vsaroz;
+
+import com.codeborne.selenide.Configuration;
+import org.testng.annotations.*;
+import com.codeborne.selenide.SelenideElement;
+
+import static com.codeborne.selenide.Condition.*;
+import static com.codeborne.selenide.Selenide.$x;
+import static com.codeborne.selenide.Selenide.open;
+
+public class SignUpTest {
+
+    private String firstName = "UsersFirstName";
+    private String lastName = "YourLastName";
+    private String email = "test.testadmin@gmail.com";
+    private String userName = "YourUserName";
+    private String password = "987654321";
+
+    private final SelenideElement Join = $x("//a[@href='/join']");
```

Comment:
all these variables are elements, rename all as I suggest it, and all variables and methods are starting from lower case letter and using CamelCase naming approach